src/core/meshcop/tcat_agent.hpp

```diff
@@ -317,6 +333,16 @@ class TcatAgent : public InstanceLocator, private NonCopyable
      */
     bool IsCommandClassAuthorized(CommandClass aCommandClass) const;
 
+    /**
+     * Indicates whether or not a command class is authorized.
```

Comment:
description to be updated to `GetAdvertisementData`

Reply:
Fixed.